sklearn/linear_model/tests/test_quantile.py

```diff
@@ -250,3 +251,36 @@ def test_linprog_failure():
     msg = "Linear programming for QuantileRegressor did not succeed."
     with pytest.warns(ConvergenceWarning, match=msg):
         reg.fit(X, y)
+
+
+@pytest.mark.skipif(
+    sp_version <= parse_version("1.6.0"),
+    reason="Solvers are available as of scipy 1.6.0",
+)
+@pytest.mark.parametrize(
+    "sparse_format", [sparse.csc_matrix, sparse.csr_matrix, sparse.coo_matrix]
+)
+@pytest.mark.parametrize("solver", ["highs"])
```

Comment:
```suggestion
@pytest.mark.parametrize("solver", ["highs-ds", "highs-ipm", "highs", "interior-point"])
```

Reply:
for interior-point solver alone, the test cases are failing. can we allow only highs* solver as of now?

Reply:
Yes. But then we should raise a ValueError, if interior-point is used with sparse data and the docstring has to be updated.

Reply:
And add it to `test_compatible_solver_sparse`.

Reply:
error persist even if I set  `'sparse': True` in solver options.

Reply:
sure, will do.